kornia/contrib/models/rt_detr/model.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from enum import Enum
+
+from kornia.contrib.models import DetectionResults
+from kornia.contrib.models.base import ModelBase
+from kornia.contrib.models.rt_detr.architecture.hybrid_encoder import HybridEncoder
+from kornia.contrib.models.rt_detr.architecture.resnet_d import ResNetD
+from kornia.contrib.models.rt_detr.architecture.rtdetr_head import RTDETRHead
+from kornia.core import Tensor, concatenate, tensor
+
+
+class RTDETRModelType(Enum):
+    r50 = 0
```

Comment:
Can we have more explicit names here and add comments?

Reply:
what should the names be? `resnet50` and `resnet101`? How about the L and X variants (HGNetv2)?